bluesky/magics.py

```diff
@@ -110,6 +114,27 @@ def ct(self, line):
     positioners = []
     FMT_PREC = 6
 
+    @line_magic
+    def detectors(self, line):
+        ''' List all available detectors.'''
+        devices = _which_devices(cls_whitelist=[ADBase], cls_blacklist=None)
+        cols = ["Python name", "Ophyd Name"]
+        print("{:20s} \t {:20s}".format(*cols))
+        print("="*40)
+        for name, obj in devices:
+            print("{:20s} \t {:20s}".format(name, obj.name))
+
+    @line_magic
+    def motors(self, line):
```

Comment:
This duplicates the functionality of `wa` below.

Reply:
thanks, we'll need to combine them